index.html

```diff
@@ -1,43 +1,80 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8" />
-        <title>My personal website</title>
-    </head>
-    <body>
-        <header></header>
-        <h1>Welcome to my personal website</h1> 
-        <nav>
-            <ul>
-                <li>Home</li>
-                <li>
-                    <a href="About Me">About Me</a>
-                </li>
-                <li>Contact</li>
-                <li>Career</li>
-            </ul>
-            </nav>
-        </header>
-        <main>
-            <section>
-                <h2>About me</h2>
-                <img
-                src="/images/Passbild_Alexander_Schulte.jpg" alt="Alex profile picture">
-                <br>
-                <a href="https://github.com/Alexander-Schulte">ty my GitHub profil</a>
-                <p>
-                    elrmvlekv lkfmvlkmrwklrmvrlkmvkrmvwkrv
-                </p>
-            </section>
-            <section>
-                <h2>Things I like to do</h2>
-                <ul>
-                    <li>Football</li>
-                    <li>Cooking</li>
-                    <li>Running</li>
-                </ul>
-            </section>
-        </main>
-        <footer>made with web technology</footer>
-    </body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <title>Alexander Schulte</title>
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to the website of Alexander Schulte</h1>
+      <nav>
+        <ul>
+          <li>Home</li>
+          <li>Contact</li>
```

Comment:
I see these 2 navigation elements are not links, compared to the ones below. Is that intended?